src/shopsync.lua

```diff
@@ -71,13 +69,40 @@ while true do
                 nbt = product.nbt,
                 displayName = product.displayName
             },
+            dynamicPrice = false,
             stock = ctx.storage.getCount(product.id, product.nbt),
             madeOnDemand = false,
             requiresInteraction = false
         })
     end
 
     -- Transmit & wait
-    txModem.transmit(BROADCAST_CHANNEL, os.getComputerID(), txMsg)
-    sleep(BROADCAST_INTERVAL_SEC)
+    txModem.transmit(BROADCAST_CHANNEL, os.getComputerID() % 65536, txMsg)
+end
+
+-- Wait for chests to be indexed
+os.pullEvent("kristify:storageRefreshed")
+
+-- Wait 15-30s before inital broadcast
+math.randomseed(os.epoch())
+os.startTimer(15 + (math.random() * 15))
+os.pullEvent("timer")
```

Comment:
I didn't think about that... that could possibly cause some issues. According to [tweaked.cc](https://tweaked.cc/module/_G.html#v:sleep) it looks like I can just swap this out with sleep because this is basically what sleep does under the hood.

Reply:
True, but if that's what sleep() is already doing under the hood, I feel like using sleep is easier/better